guides/doc-Managing_Hosts/topics/proc_registering-a-host.adoc

```diff
@@ -39,86 +39,106 @@ Optional: If the *Registration* feature is not enabled on your {SmartProxy}, ent
 
 .Procedure
 . Navigate to *Hosts* > *Register Host*.
+. Optional: Select a different *Organization*.
+. Optional: Select a different *Location*.
 . Optional: From the *Host Group* list, select the host group to associate the hosts with.
-Fields that inherit value from *Host group*: Operating System, Activation Key(s) and Lifecycle environment.
-. From the *Operating System* list, select the operating system of hosts that you want to register.
-. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through. You must select the internal {SmartProxy} if you do not want to use an external {SmartProxy}.
-. Optional: *Insecure* - This makes the first call insecure, however, during this first call, hosts download the CA file from {Project}. Hosts will use this CA file to connect to {Project} with all future calls making them secure.
+Fields that inherit value from *Host group*: *Operating system*, *Activation Keys* and *Lifecycle environment*.
+. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through.
+ifdef::satellite[]
+You must select the internal {SmartProxy} if you do not want to use an external {SmartProxy}.
+endif::[]
+. Optional: From the *Operating system* list, select the operating system of hosts that you want to register.
+ifndef::satellite[]
+Specifying an operating system is required when you register machines without `subscription-manager`, such as Debian or Ubuntu.
+endif::[]
+. Optional: Select the *Insecure* option, if you want to make the first call insecure.
+During this first call, hosts download the CA file from {Project}.
+Hosts will use this CA file to connect to {Project} with all future calls making them secure.
++
+It is recommended to avoid insecure calls.
+Instead, you can manually copy and install the CA file on each host before registering the host.
++
+To do this, find where {Project} stores the CA file by navigating to *Administer* > *Settings* > *Authentication* and locating the value of the *SSL CA file* setting.
++
+Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on hosts and enter the following commands:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# update-ca-trust enable
+# update-ca-trust
+# curl -sS https://{foreman-example-com}/register ...
+----
 +
 If an attacker, located in the network between {Project} and a host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from the registered host and the JSON Web Tokens (JWT).
 Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
```

Comment:
I suggest to move "To do this" after this paragraph.

Then break it up into two snippets: one with the update-ca-trust commands (both) and one with the curl command. That should be divided by some filler, something like "To register a host, use the following command".

Reply:
To elaborate this more (I had a meeting sorry), update-ca-trust and curl is one way to do it, the other way is just insecure curl. So we want these two snippets, while I mentioned breaking up the one on line 66-70, that is actually not necessary. Depending on how you like, you might or might not break it I do not insist. It is just we need to have the curl command twice (with and without the insecure option).

I hope it makes sense.

Reply:
Got it.
However, I think that the "To do this" paragraph would better fit right after "It is recommended", otherwise the flow doesn't make sense IMHO. Check it out in changed files.